tests/fixtures/modules/import/error-strict.src.js

```diff
@@ -0,0 +1,5 @@
+import house from "house";
```

Comment:
I was thinking `import` was needed to somehow trigger module mode, but obviously that's not the case. Either way :)